### ISSUE TITLE
In compatible examples/documentation

### ISSUE THREAD

**Issue by Shubhamcl** [carla-simulator/carla#3744]
As per current state of the code, where is the documentation to actually get started.

I can see that currently there are steps to run the simulator in spectator mode, and then tutorials directly start from adding friction.

While the previous versions have some documentations, the current examples don't work when directly ran according to the previous documentations.

Have the examples gone stale? or is some step which integrates them with the current state of carla missing in the documentation?

EDIT:
Basically even after somehow following all non-build instructions, installing the Python carla api and being able to import the functionality into python scripts, this is still seen when running the examples:

Traceback (most recent call last):
  File "spawn_npc.py", line 26, in <module>
    from carla import VehicleLightState as vls
ImportError: cannot import name 'VehicleLightState' from 'carla' (/home/xxxxxx/anaconda3/envs/carlaenv/lib/python3.7/site-packages/carla/__init__.py)

**Comment by corkyw10** [comment]
Hi @Shubhamcl are you still facing this issue? These problems often arise from using different versions of Python, did you install the API and dependencies with python 3.7?
